BookWorm/BookWorm/TableViewCells/InnerChatCell.m

```diff
@@ -0,0 +1,23 @@
+//
+//  InnerChatCell.m
+//  BookWorm
+//
+//  Created by Ramya Prabakar on 7/20/22.
+//
+
+#import "InnerChatCell.h"
+
+@implementation InnerChatCell
+
+- (void)awakeFromNib {
+    [super awakeFromNib];
+    // Initialization code
```

Comment:
nit: I assume this is xocde generated comment, maybe we just remove it instead of committing into codebase